_src/modules/data.js

```diff
@@ -0,0 +1,209 @@
+import * as d3 from 'd3';
+import {getLayoutVars} from './layout_vars';
+
+var data=[];
+var lobby=[];
+var lobby_level_0=[];
+var myGroups = new Set();
+var myArrGroups = new Array();
+var isSF = getLayoutVars().type === 'sf';
+var prefix = getLayoutVars().isProd ? 'https://declarator.org/media/dumps/' : '/assets/data/';
+var files = {
+    "duma-7": [
+        prefix + 'lobbist-small-d7.json',
+        prefix + 'lobby-group.json',
+        "/assets/data/rating.json",
+        "/assets/data/alias.json",
+    ],
+    "duma-8": [
+        prefix + 'lobbist-small-d8.json',
+        prefix + 'lobby-group.json',
+        "/assets/data/rating.json",
+        "/assets/data/alias.json",
+    ],
+    "sf": [
+        prefix + 'lobbist-small-sf.json',
+        prefix + 'lobby-group.json',
+        "/assets/data/rating-sf.json",
+        "/assets/data/alias.json"
+    ],
+}[getLayoutVars().type];
+
+var promises = [];
+var rawDep, rawRating, rawAlias, rawSF, aliases;
+
+files.forEach(function (url) {
+    promises.push(d3.json(url))
+});
+
+var cors="https://cors-anywhere.herokuapp.com/"
+var url_lobby="https://dev.declarator.org/api/lobby_group/"
+//promises.push(getAPI(allData,null,url_lobby));
+
+
+export default Promise.all(promises).then(function (values) {
+    //console.log("file1", values[0]) //dep
+    rawDep = values[0]
+    var rawLobby = values[1] //change to load from url
+    rawRating = values[2] //
+    rawAlias = values[3]
+    dataDepMap(rawDep)
+    getAliasMap(rawAlias)
+    getLobbyMap(rawLobby,aliases)
+
+    return {
+        lobby: lobby,
+        lobby_level_0: lobby_level_0,
+        data: data,
+        rawRating: rawRating,
+        rawDep: rawDep,
+        myGroups: myGroups,
+        myArrGroups: myArrGroups,
+        isSF:isSF
+    }
```

Comment:
в качестве улучшения читаемости можно сократить запись
```suggestion
    return {
        lobby,
        lobby_level_0,
        data,
        rawRating,
        rawDep,
        myGroups,
        myArrGroups,
        isSF
    }
```